tests/test_cliCmdTestIngest.py

```diff
@@ -32,59 +32,75 @@ class IngestRawsTestCase(CliCmdTestBase, unittest.TestCase):
 
     @staticmethod
     def defaultExpected():
-        return dict(directory=None,
-                    file=None,
-                    transfer="auto",
+        return dict(config={},
+                    config_file=None,
                     ingest_task="lsst.obs.base.RawIngestTask",
-                    config={},
-                    config_file=None)
+                    locations=(),
+                    output_run=None,
+                    regex=None,
+                    transfer="auto")
 
     @staticmethod
     def command():
         return ingest_raws
 
     def test_repoAndOutput(self):
         """Test the most basic required arguments, repo and output run"""
-        self.run_test(["ingest-raws", "here",
+        self.run_test(["ingest-raws", "repo", "files",
```

Comment:
Hmm, maybe "resources" rather than "files" (but I'm okay with "files" since all the remote resources will be files eventually).